packages/web-components/src/components/card/card.scss

```diff
@@ -18,6 +18,14 @@
     color: $inverse-link;
   }
 
+  &[color-scheme='light'] {
```

Comment:
does the static card always have the `light` color-scheme? what happens if we attach the styles to 

```
:host(#{$dds-prefix}-card-footer),
:host(#{$dds-prefix}-card-cta-footer)
```

Reply:
@annawen1, yes it's always on in static. Can't find another way to target static card. Tested it on other variations and they all work fine except for the light schem.